packages/connect/src/data/config.ts

```diff
@@ -72,6 +72,11 @@ export const config = {
             download: 'https://www.google.com/chrome/',
             update: 'https://support.google.com/chrome/answer/95414',
         },
+        mobilechrome: {
```

Comment:
@komret I am not particuolary sure about this.

The new UA parser returns android as `mobilechrome` not  as `chrome` as the earlier versions.

So, the fix here is whacky here. Maybe we could say that if ua parser returns browser name as `mobilechrome` we could treat it as `chrome` but.

But this seems cleaner to me. Without it, the tests logically fail.